cypress/support/fragments/organizations/organizations.js

```diff
@@ -650,14 +650,18 @@ export default {
     );
   },
 
-  deleteOrganization: () => {
+  deleteOrganization: (confirm = false) => {
     cy.do([
       PaneHeader({ id: 'paneHeaderpane-organization-details' }).find(actionsButton).click(),
       Button('Delete').click(),
-      Button({
-        id: 'clickable-delete-organization-confirmation-confirm',
-      }).click(),
     ]);
+    if (confirm === false) {
```

Comment:
I'd recommend to change this condition - to perform click action confirm should be "true"